modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/resolver/OrtScannerResultResolver.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.ort.resolver;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.eclipse.sw360.antenna.model.artifact.Artifact;
+import org.eclipse.sw360.antenna.model.artifact.facts.ArtifactSourceUrl;
+import org.eclipse.sw360.antenna.model.artifact.facts.CopyrightStatement;
+import org.eclipse.sw360.antenna.model.artifact.facts.ObservedLicenseInformation;
+import org.eclipse.sw360.antenna.util.LicenseSupport;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Optional;
+
+public class OrtScannerResultResolver extends AbstractOrtResultResolver {
+    public OrtScannerResultResolver(JsonNode result) {
+        super(result);
+    }
+
+    @Override
+    public Artifact mapArtifact() {
+        Artifact a  = super.mapArtifact();
+        mapCopyrights().ifPresent(a::addFact);
+        mapObservedLicense().ifPresent(a::addFact);
+        return a;
+    }
+
+    @Override
+    public Optional<ArtifactSourceUrl> mapSourceUrl() {
+        String sourceUrl;
+
+        Iterator<JsonNode> ortItemProvenanceIterator = result.get("results").elements();
+        if (ortItemProvenanceIterator.hasNext()) {
+            JsonNode provenanceNode = ortItemProvenanceIterator.next().get("provenance");
+            if (provenanceNode.has("vcs_info")) {
+                sourceUrl = provenanceNode.get("vcs_info").get("url").textValue();
+            } else if (provenanceNode.has("source_artifact")) {
+                sourceUrl = provenanceNode.get("source_artifact").get("url").textValue();
+            } else if (provenanceNode.has("original_vcs_info")) {
+                sourceUrl = provenanceNode.get("original_vcs_info").get("url").textValue();
+            } else {
+                return Optional.empty();
+            }
+            return Optional.of(new ArtifactSourceUrl(sourceUrl));
+        }
+        return Optional.empty();
+    }
+
+    private Optional<ObservedLicenseInformation> mapObservedLicense() {
+        if(result.get("results").isArray()) {
+            Collection<String> licenses = new ArrayList<>();
+            Iterator<JsonNode> results = result.get("results").elements();
+            while(results.hasNext()) {
+                Iterator<JsonNode> findings = results.next().get("summary").get("license_findings").elements();
```

Comment:
duplicated code, same line as in https://github.com/eclipse/antenna/pull/199/files#diff-5363d56ff7d691e5d496f9256fd61b73R82